src/test/regress/input/multi_subquery.source

```diff
@@ -294,3 +294,31 @@ FROM
 		l_orderkey = o_orderkey
 	GROUP BY
 		l_orderkey) AS unit_prices;
+
+-- Check that we can prune shards in subqueries with VARCHAR partition columns
+
+CREATE TABLE subquery_pruning_varchar_test_table
+(
+   a varchar,
+   b int
+);
+
+SELECT master_create_distributed_table('subquery_pruning_varchar_test_table', 'a', 'hash');
+SELECT master_create_worker_shards('subquery_pruning_varchar_test_table', 4, 1);
+
+SET citus.subquery_pushdown TO TRUE;
+SET client_min_messages TO DEBUG2;
+
+SELECT * FROM 
+	(SELECT count(*) FROM subquery_pruning_varchar_test_table WHERE a = 'onder' GROUP BY a) 
+AS foo;
+
+SELECT * FROM 
+	(SELECT count(*) FROM subquery_pruning_varchar_test_table WHERE 'eren' = a GROUP BY a)
+AS foo;
+
+SET client_min_messages TO INFO;
```

Comment:
I just realized that the default value of `client_min_messages` is `NOTICE`. Could you please check whether do we need `NOTICE` or `INFO` here (i.e., What is the value of `client_min_messages` before it is set to `DEBUG2`)?